ansible/_prereqs.yaml

```diff
@@ -0,0 +1,18 @@
+---
+
+  # Force fact gathering once the nodes have been bootstrapped
+  - hosts: node control-plane
+    name: "Gather Node Facts"
+    gather_facts: yes
+    tasks: []
+
+  - hosts: node control-plane
+    any_errors_fatal: true
+    name: "Configure Cluster Prerequisites"
+    become: yes
+    environment: "{{ combined_env }}"
+
+    roles:
+      - prereqs
+      - disable-swap
+      - cloud-netconfig-manage
```

Comment:
Isn't this already handled somewhere in this repo? Other Ansible tasks are being already run.